### ISSUE TITLE
MP4Box "Bad SRT formatting" when upgrading to 1.0.0

### ISSUE THREAD

**Issue by mweber03** [gpac/gpac#1536]
To date I had been using `0.5.1-DEV-rev5568` and I noticed today that I was way behind on versions. I decided to try upgrading, so I installed 1.0.0. When trying to use the new version, I got many `[TXTIn] Bad SRT formatting - expecting number got...` and `[TXTIn] Corrupted SRT frame X after frame Y`. And many, but not all, lines of the subtitle file get skipped. This does not happen using the same files in the v0.5.1 I was using before. The command I was running (for both versions) was: `mp4box.exe -add "My Subs.srt":hdlr=sbtl:lang=eng "My Movie.mp4" -out "My Movies with subs.mp4"`.

I'm running on Windows x64. I've attached one SRT file and its associated command line outputs when running the above command. I tried many other SRTs and movies with the same issues. I also tried multiple versions of 1.0.0: `1.0.0-rev0-ga99c032b-master` and `1.0.0-rev87-g0bfe309f-master` (the latest nightly build) with the same result.

Let me know if you require any additional information.

[My Subs.srt](https://github.com/gpac/gpac/files/4875648/My.Subs.srt.txt)
[output.txt](https://github.com/gpac/gpac/files/4875649/output.txt)

**Comment by mweber03** [comment]
It seems to also work fine for `0.8.1-rev6-gb54511e7-legacy`.

**Comment by jeanlf** [comment]
Hmm I cannot reproduce here:
```
% MP4Box
Not enough arguments - check usage with -h
MP4Box - GPAC version 1.0.0-rev87-g0bfe309fc-master
(c) 2000-2020 Telecom Paris distributed under LGPL v2.1+ - http://gpac.io

% MP4Box -add "My Subs.srt":hdlr=sbtl:lang=eng "My Movie.mp4" -out "My Movies with subs.mp4"
Track Importing Timed Text - Text track 1280 x 720 font Serif (size 18) layer 0
0.500 secs Interleaving  

```

Which OS are you running ?

**Comment by mweber03** [comment]
Windows 10 x64

**Comment by mweber03** [comment]
Tried again with the latest nightly build: `1.0.0-rev92-g0cb878df-master` (downloaded from [gpac-1.0.0-rev92-g0cb878df-master-x64.exe](https://download.tsi.telecom-paristech.fr/gpac/new_builds/win64/gpac-1.0.0-rev92-g0cb878df-master-x64.exe))

Still happens:
```
% mp4box
Not enough arguments - check usage with -h
MP4Box - GPAC version 1.0.0-rev92-g0cb878df-master
(c) 2000-2020 Telecom Paris distributed under LGPL v2.1+ - http://gpac.io

% mp4box -add "My Subs.srt":hdlr=sbtl:lang=eng "My Movie.mp4" -out "My Movies with subs.mp4"
Track Importing Timed Text - Text track 1920 x 800 font Serif (size 18) layer 0
[TXTIn] Bad SRT formatting - expecting number got ":24:21,294 --> 00:24:23,630"
[TXTIn] Bad SRT formatting - expecting number got "Hurry, he's in the escape pod."
[TXTIn] Corrupted SRT frame 3 after frame 1
[TXTIn] Bad SRT formatting - expecting number got ":41,684 --> 00:26:43,686"
[TXTIn] Bad SRT formatting - expecting number got "They're landing."
[TXTIn] Corrupted SRT frame 5 after frame 3
[TXTIn] Bad SRT formatting - expecting number got ":57,005 --> 00:29:58,965"
[TXTIn] Bad SRT formatting - expecting number got "Let's get him."
[TXTIn] Corrupted SRT frame 7 after frame 5
[TXTIn] Bad SRT formatting - expecting number got ":10,351 --> 00:30:13,104"
[TXTIn] Bad SRT formatting - expecting number got "The president is no longer onboard."
[TXTIn] Corrupted SRT frame 9 after frame 7
[TXTIn] Corrupted SRT frame 3 after frame 9
[TXTIn] Error scanning SRT frame 3 timing
[TXTIn] Corrupted SRT frame 11 after frame 3
[TXTIn] Corrupted SRT frame 4 after frame 11
[TXTIn] Error scanning SRT frame 4 timing
[TXTIn] Corrupted SRT frame 13 after frame 4
...
```

**Comment by jeanlf** [comment]
should now be fixed, could you cross-check ?

**Comment by mweber03** [comment]
It's fixed! Tested with `1.0.0-rev103-g925bae1e-master`. Thank you.
